tools/slo-generator/tests/unit/test_cli.py

```diff
@@ -15,22 +15,55 @@
 import os
 import unittest
 
-from slo_generator.cli import parse_args
+from mock import patch
+from slo_generator.cli import parse_args, cli
+from test_stubs import CTX, mock_sd
 
 cwd = os.path.dirname(os.path.abspath(__file__))
+root = os.path.dirname(os.path.dirname(cwd))
 
 
 class TestCLI(unittest.TestCase):
 
+    def setUp(self):
+        for k, v in CTX.items():
+            os.environ[k] = v
+
     def test_parse_args(self):
         slo_config_path = f'{cwd}/fixtures/slo_config.json'
         error_budget_policy_path = f'{cwd}/fixtures/error_budget_policy.json'
         args = parse_args([
-            "--slo-config", slo_config_path, "--error-budget-policy",
-            error_budget_policy_path
+            '--slo-config', slo_config_path, '--error-budget-policy',
+            error_budget_policy_path, '--export'
         ])
         self.assertEqual(args.slo_config, slo_config_path)
         self.assertEqual(args.error_budget_policy, error_budget_policy_path)
+        self.assertEqual(args.export, True)
+
+    @patch('google.api_core.grpc_helpers.create_channel',
+           return_value=mock_sd(8))
+    def test_cli(self, mock):
+        args = parse_args([
+            '-f', f'{root}/samples/stackdriver/slo_gae_app_availability.yaml',
+            '-b', f'{root}/samples/error_budget_policy.yaml'
+        ])
+        cli(args)
```

Comment:
Can we assert anything about this?

Reply:
The `cli` method doesn't return anything, so I need to think more about what we could check here. Maybe that we are calling the `compute` method exactly one time ?

Reply:
Ok, I've added a return value to `cli`: it now returns a dictionary of all reports indexed by file path. 

Added two checks for each test:
* Check that the filepath passed is present in the SLO reports dictionary. 
* Check that first batch contains 4 reports, one per step in the error budget policy.